tecken/tests/test_download.py

```diff
@@ -37,164 +37,143 @@ def reload_downloaders(urls, try_downloader=None):
         views.try_downloader = SymbolDownloader([try_downloader])
 
 
-def test_client_happy_path(client, botomock):
-    reload_downloaders("https://s3.example.com/private/prefix/")
+def test_client_happy_path(client, db, s3_helper):
```

Comment:
A lot of these tests relied on the private bucket support which was botomocked out. In order to remove private bucket support, I had to rewrite the download tests to not use botomock.